open_schools_platform/organization_management/organizations/models.py

```diff
@@ -32,8 +32,7 @@ def query_handler(query: Query, new_status: str):
         if query.status == Query.Status.ACCEPTED:
             query.body.organization = query.sender  # type: ignore
             query.body.employee_profile = query.recipient  # type: ignore
-
-        query.body.save()  # type: ignore
+            query.body.save()  # type: ignore
```

Comment:
add if wiht such expression
1) sender - Organization
2) recipient - Employee

it is for skipping queries with others parametrs
in "wrong" case we should raise 400 exception